linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpConfig.scala

```diff
@@ -35,6 +34,7 @@ class HttpInitializer extends ProtocolInitializer.Simple {
       .prepend(Headers.Dst.BoundFilter.module)
     val clientStack = Http.router.clientStack
       .prepend(http.AccessLogger.module)
+      .prepend(HttpLoggerConfig.module)
```

Comment:
I think we want this to be somewhere in the endpoint stack so that is has access to the endpoint addr.  How about 
```
.insertAfter(ClientDestTracingFilter.Role, HttpLoggerConfig.module)
```